docs/terminal/amberleyoakey.md

```diff
@@ -4,7 +4,103 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
 
+--8<-- "includes/abbreviations.md"
+
+## YAMB Positions
+
+| Name               | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| Amberley TCU   | Amberley Approach   | 126.200        | AM_APP                                   |
+
+
+
+## Airspace
+### AM TCU
+#### Restricted Areas
+By Default Amberley owns all of the R625 Restricted Areas, detailed below:
+
+- R625A (`A015`-`A085`)  
+- R625B (`A025`-`A085`)  
+- R625C (`A045`-`A085`)  
+- R625D (`A085`-`F210`)  
+
+### Optional, as required
+AM APP can negotiate further airspace releases from surrounding ENR sectors of the following Restricted Areas:
+
+- R650A (`A050`-`A100`)  
+- R650B (`A050`-`F600`)  
+- R662A (`A050`-`F600`)  
+- R662B (`A050`-`F600`)  
+- R671A (`F125`-`F245`)  
+- R671B (`F240`-`F600`)
+
+!!! note
+    When Controlling a Military Aerodrome on VATSIM its important to consider any notams regarding VSOAs as Restricted Airspaces may be activated. [VATPAC Notams](https://vatpac.org/publications/notam){target=new}
```

Comment:
'NOTAMs' should be capitilised. Plural of airspace is airspace (not airspaces). Maybe expand the wording around the link similar to 'See [VATPAC NOTAMs] for active NOTAMs which may affect military operations.'